Automattic/woocommerce-payments
# Add transaction list sorting

Fixes #595 

#### Changes proposed in this Pull Request

* Add sorting to date, amount, fees and net
* Migrate transaction list tests to rtl

#### Testing instructions



* Navigate to the transactions list and make sure it is sorted by date descending
* Check that the info button background matches the date column
* Ensure you are able to sort by those fields by clicking in the header

![image](https://user-images.githubusercontent.com/7714042/79615484-4ad5b700-80d9-11ea-974b-08d845e01b59.png)

![image](https://user-images.githubusercontent.com/7714042/79615493-50330180-80d9-11ea-9fa4-1e455eb023cd.png)

-------------------
